sdk/simulator-service/src/implementation/helpers/index.ts

```diff
@@ -0,0 +1,61 @@
+import { TokenAmount, type Token } from '@summerfi/sdk-common/common/implementation'
+import type {
+  ReferencableField,
+  SimulationStrategy,
+  ValueReference,
+} from '@summerfi/sdk-common/simulation'
+import type { Tail } from '~swap-service/interfaces/helperTypes'
+
+export function makeStrategy<T extends SimulationStrategy>(strategy: T): T {
+  return strategy
+}
+
+export function isReference<T>(value: ReferencableField<T>): value is ValueReference<T> {
```

Comment:
`isValueReference`? I think it is easier to understand that ValueReference and isValueReference are connected. 

I think the correct spelling for `ReferencableField` is `ReferenceableField` (notice the extra `e` between the `c` and the `a`)